packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -51,11 +52,14 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
     });
     const invitee = await getUserToInviteOrThrowIfExists({
       usernameOrEmail,
-      orgId: input.teamId,
       isOrg: input.isOrg,
     });
 
     if (!invitee) {
+      const isInputEmail = isEmail(usernameOrEmail);
+      if (!isInputEmail) {
+        throwUsernameDoesNotExistError(usernameOrEmail);
+      }
```

Comment:
This doesn't make too much sense. Throw that user doesn't exist without even search for it doesn't make too much sense for me 🤔